cnxepub/formatters.py

```diff
@@ -191,7 +191,7 @@ def __str__(self):
     def __bytes__(self):
         if not self.built:
             self.build()
-        return etree.tostring(self.root, pretty_print=True)
+        return etree.tostring(self.root, pretty_print=True, method='html')
```

Comment:
This should fix the weird nesting problem because the browser doesn't see the file as xhtml if the file is saved with extension `.html`

![screen shot 2016-03-31 at 16 13 25](https://cloud.githubusercontent.com/assets/153353/14229338/cbdcf832-f8f6-11e5-9a6a-5492725af7bd.png)